Fix invalid syntax in example-config.json

PeterChen13579 opened the pull request:
remove "." fixes syntax for example-config; runs with ./clio_server

PeterChen13579 committed:
Update example-config.json

remove "." so that example-config works runs with clio server

kuznetsss commented on docs/examples/config/example-config.json:
```diff
@@ -67,7 +67,7 @@
         "local_amdin": false
     },
     // Time in seconds for graceful shutdown. Defaults to 10 seconds. Not fully implemented yet.
-    "graceful_period": 10.,
```
It would be better to change it to `10.0` to show that it is floating point value

PeterChen13579 committed:
Update example-config.json

change "graceful_period" value from 10 to 10.0 to show it is floating point value

kuznetsss reviewed (approved):
Thanks for the fix 👍
